enterprise/migrations/0141_make_enterprisecatalogquery_title_unique.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 2.2.24 on 2021-08-27 17:56
+
+from django.db import migrations, models
+
+def force_unique_title(apps, schema_editor):
+    # I've verified there are no duplicates in Stage and Prod.
+    # This was brought up as a thing devs might need for their local devstack.
```

Comment:
Thank you for this note, and for thinking of dev environments (or maybe Open edX environments?).

Reply:
Alex S was the one to note this and yeah our thought was development, but definitely also may apply to Open edX!